### DISCUSSION TITLE
Section 3 - Terraform básico] - Editor de texto

### DISCUSSION THREAD

**Discussion by DiegoMedeirosLima** [chgasparoto/curso-aws-com-terraform#74] [Q&A]
Onde está editando o script que copiou (Criação do bucket)?
Onde faço essa edição?
Pela leitura parece ser no git por conta do gitignore, mas não sei como ou quais opções configura na criação de um projeto, isso se eu estiver no caminho certo.

Essa é minha dúvida.

**Comment by chgasparoto** [comment]
Olá @DiegoMedeirosLima 

> Onde está editando o script que copiou (Criação do bucket)?

Eu criei outra pasta chamada [02-alterando-destruindo](https://github.com/chgasparoto/curso-aws-com-terraform/tree/master/01-terraform-basico/02-alterando-destruindo) no mesmo nível da pasta [01-primeiro-script](https://github.com/chgasparoto/curso-aws-com-terraform/tree/master/01-terraform-basico/01-primeiro-script) .

> Pela leitura parece ser no git por conta do gitignore, mas não sei como ou quais opções configura na criação de um projeto, isso se eu estiver no caminho certo.

Não entendi muito bem esta dúvida mas para criar um "projeto" para o Terraform, basta você colocar todos os arquivos, com extensão **_.tf_** , necessários juntos dentro de uma mesma pasta.

Abraço.
